application/F3DOptionsTools.cxx

```diff
@@ -214,21 +222,23 @@ void PrintHelp(const std::string& execName, const cxxopts::Options& cxxOptions)
   orderedCLIGroupNames.reserve(::CLIOptions.size());
   for (const ::CLIGroup& optionGroup : ::CLIOptions)
   {
-    // This ensure help is provided in the expected group order
     orderedCLIGroupNames.emplace_back(optionGroup.GroupName);
   }
+  
   f3d::log::info(cxxOptions.help(orderedCLIGroupNames));
+  
   f3d::log::info("Keys:");
   for (const auto& [key, desc] : f3d::interactor::getDefaultInteractionsInfo())
   {
-    ::PrintHelpPair(key, desc);
+    ::PrintHelpPair(key, desc, 20);
   }
 
   f3d::log::info("\nExamples:");
   for (const auto& [cmd, desc] : examples)
   {
-    ::PrintHelpPair(cmd, desc, 50);
+    ::PrintHelpPair(cmd, desc, 5);
```

Comment:
why changing this ?

Reply:
It makes the text more close to each other.